src/components/application_manager/test/CMakeLists.txt

```diff
@@ -41,6 +41,8 @@ include_directories(
 #=============== COMMANDS =====================
 set(COMMANDS_PATHS
   ${CMAKE_CURRENT_SOURCE_DIR}/commands
+  ${CMAKE_CURRENT_SOURCE_DIR}/commands/hmi
```

Comment:
@VVeremjova you should not specify subdirectories. They will be included by default